reference/core/ambassador.md

```diff
@@ -177,11 +177,13 @@ If set, `cluster_idle_timeout_ms` specifies the timeout (in milliseconds) after
 
 ### gRPC HTTP/1.1 bridge (`enable_grpc_http11_bridge`)
 
-Ambassador Edge Stack supports bridging HTTP/1.1 clients to backend gRPC servers. When an HTTP/1.1 connection is opened and the request content type is `application/grpc`, Ambassador Edge Stack will buffer the response and translate into gRPC requests. For more details on the translation process, see the [Envoy gRPC HTTP/1.1 bridge documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_http1_bridge_filter). This setting can be enabled by setting `enable_grpc_http11_bridge: true`.
+Ambassador Edge Stack supports bridging HTTP/1.1 clients to backend gRPC servers. When an HTTP/1.1 connection is opened and the request content type is `application/grpc`, Ambassador Edge Stack will buffer the response and translate into gRPC requests. For more details on the translation process, see the [Envoy gRPC HTTP/1.1 bridge documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/grpc_http1_bridge_filter). This setting can be enabled by setting `enable_grpc_http11_bridge: true`. Read more about [gRPC and Ambassador Edge Stack](user-guide/grpc).
```

Comment:
will the link to the user-guide/grpc work as is now? aka, we aren't using the ../.. anymore?

Reply:
You're right, and it's broken in the currently-live version of the GA docs too.  I guess I'll do some work on the broken link checker.

Reply:
I've fixed this separately in https://github.com/datawire/ambassador-docs/pull/172 (EA) and https://github.com/datawire/ambassador-docs/pull/173 (GA)